### PR TITLE
docs: serialisable vs non-serialisable

### LINKED ISSUE #151: Add Scilla Summary page for quick reference
Here is an initial stub by @noel-yoo 

### Serialisable:
- Byte strings are serialisable.
- Addresses are serialisable only as `ByStr20` values.
- All other primitive types like integers and strings are serialisable.
- ADT are serialisable if the types of their subvalues are serialisable. This means that the type of every constructor argument must be serialisable.

### Not Serialisable:
- Messages, events and the special `Unit` type are not serialisable.
- Function types and map types are not serialisable.
- Complex types involving uninstantiated type variables are not serialisable.

## Storable types

TODO

### PR DESCRIPTION
**Pull request by noel-yoo**
closes #151

### CODE REVIEW

File: docs/source/scilla-in-depth.rst [hunk 1]
```diff
@@ -294,23 +294,25 @@ multiple parameters are separated by ``,``.
...
+    
+    - Byte strings are serialisable.
 
-   - Byte strings are serialisable. Addresses are serialisable only as
-     ``ByStr20`` values. All other primitive types like integers and
-     strings are serialisable.
+    - Addresses are serialisable only as ``ByStr20`` values.
 
-   - Function types and map types are not serialisable.
+    - All other primitive types like integers and strings are serialisable.
```
Comment (jjcnn) [inline docs/source/scilla-in-depth.rst:305]: This needs to be rephrased. Messages, events and the unit type are considered primitive types in Scilla, so they are captured by this statement, yet they are not serialisable.

**Review by jjcnn — CHANGES_REQUESTED** [review]

File: docs/source/scilla-in-depth.rst (hunk 1 above)
Comment (noel-yoo) [reply docs/source/scilla-in-depth.rst:305]: `Byte strings, Integers, strings, block numbers are serialisable.`
How about this?

Comment (jjcnn) [reply docs/source/scilla-in-depth.rst:305]: Should be `and block numbers`, but otherwise fine.

Comment (noel-yoo) [reply docs/source/scilla-in-depth.rst]: sure

**Review by jjcnn — APPROVED** [review]

**Pull request merged**